nbclient/tests/test_client.py

```diff
@@ -201,10 +204,11 @@ def normalize_output(output):
         if isinstance(value, string_types):
             output['data'][key] = normalize_base64(value)
     if 'traceback' in output:
-        tb = [
-            re.sub(ipython_input_pat, '<IPY-INPUT>', strip_ansi(line))
-            for line in output['traceback']
-        ]
+        tb = []
+        for line in output["traceback"]:
+            line = re.sub(ipython_input_pat, '<IPY-INPUT>', strip_ansi(line))
+            line = re.sub(ipython8_input_pat, '<IPY-INPUT>', strip_ansi(line))
+            tb.append(line)
```

Comment:
```suggestion
        tb = [
            re.sub(ipython8_input_pat, '<IPY-INPUT>', strip_ansi(
                re.sub(ipython_input_pat, '<IPY-INPUT>', strip_ansi(line))
            )
            for line in output['traceback']
        ]
```

Reply:
Well, honestly, my implementation seems to be more readable to me but it's your codebase so, if you want to, apply this suggestion. It might be also possible to merge the two regexes to one but again, readability would suffer.

Reply:
It was just a suggestion, I have no strong preference.